evergreen.yml

```
@@ -1495,7 +1447,9 @@ functions:
           export RUST_BACKTRACE=1
           # Start a local ADF
           $COMMON_TEST_INFRA_DIR/test-environment/run_adf.sh start &&
-          cargo run --bin data_loader &&
+          cd $COMMON_TEST_INFRA_DIR &&
+          cargo run --bin data-loader -- --mongod-uri mongodb://localhost:28017 --adf-uri mongodb://${adf_test_local_user}:${adf_test_local_pwd}@localhost:27017 -d ../resources/integration_test/testdata &&
+          cd - &&
```

```suggestion
          cargo run --manifest-path "$COMMON_TEST_INFRA_DIR/Cargo.toml" --bin data-loader -- \
            --mongod-uri mongodb://localhost:28017 \
            --adf-uri "mongodb://${adf_test_local_user}:${adf_test_local_pwd}@localhost:27017" \
            -d "$COMMON_TEST_INFRA_DIR/../resources/integration_test/testdata" &&
```
Take a look at this way, it avoids the cd'ing in and out.